contracts/TokenPool.sol

```diff
@@ -159,7 +159,7 @@ contract TokenPool is Migratable, Pausable {
         atonomiStorage.deleteBool(keccak256("network", msg.sender, "isIRNNode"));
         atonomiStorage.deleteBytes32(keccak256("network", msg.sender, "memberId"));
 
-        emit ManufacturerRewardWalletChanged(msg.sender, _new, oldMemberId);
+        emit ManufacturerRewardWalletChanged(msg.sender, _new, newMemberId);
```

Comment:
Double check the original master code: https://github.com/atonomi/smart-contracts/blob/master/contracts/Atonomi.sol#L633

Reply:
These should be equivalent according to the original code. Is it ever possible we would have a new/different memberId when changing wallet?

Reply:
No the memberid would have to be the same in this scenario, otherwise, if different, we can treat as a new added member.

Reply:
I called this out for the difference in the emitted event `emit ManufacturerRewardWalletChanged(msg.sender, _new, old.memberId);`  last parameter should be reporting the oldMemberId in the log.